pages/api/get.ts

```diff
@@ -2,14 +2,14 @@ import { VercelRequest, VercelResponse } from "@vercel/node";
 import fetch from "node-fetch";
 import { decode } from "js-base64";
 
-const get = (request: VercelRequest, response: VercelResponse) => {
-  const { code = "" } = request.query;
+const get = (req: VercelRequest, res: VercelResponse) => {
```

Comment:
<img width="481" alt="Screenshot 2021-08-08 at 8 40 42 PM" src="https://user-images.githubusercontent.com/69431456/128636771-d647ec81-eb48-4182-b1b2-435e5ec17c02.png">

I think you can use NextApiHandler instead of vercel's 🤔